### ISSUE TITLE
lmer calls rBind which throws an error

### ISSUE THREAD

**Issue by revelle** [lme4/lme4#461]
Using lmer in another package I have an error message (when checking on win-builder) that seems be a call to rBind in lmer.  

Consider the following example:

```
x.df <- structure(list(values = c(9, 6, 8, 7, 10, 6, 2, 1, 4, 1, 5, 2, 
5, 3, 6, 2, 6, 4, 8, 2, 8, 6, 9, 7), items = structure(c(1L, 
1L, 1L, 1L, 1L, 1L, 2L, 2L, 2L, 2L, 2L, 2L, 3L, 3L, 3L, 3L, 3L, 
3L, 4L, 4L, 4L, 4L, 4L, 4L), class = "factor", .Label = c("J1", 
"J2", "J3", "J4")), id = structure(c(1L, 2L, 3L, 4L, 5L, 6L, 
1L, 2L, 3L, 4L, 5L, 6L, 1L, 2L, 3L, 4L, 5L, 6L, 1L, 2L, 3L, 4L, 
5L, 6L), .Label = c("S1", "S2", "S3", "S4", "S5", "S6"), class = "factor")), row.names = c(NA, 
-24L), class = "data.frame")

mod.lmer <- lme4::lmer(values ~ 1 + (1 | id) + +(1 | items), 
    data = x.df, na.action = na.omit)
```
This leads to rBind being called.
```
debug(Matrix::rBind)
mod.lmer <- lme4::lmer(values ~ 1 + (1 | id) + +(1 | items), 
    data = x.df, na.action = na.omit)
```

**Comment by bbolker** [comment]
`rBind` has been deprecated since R version 3.2.0.  Should we try to maintain backward compatibility?

**Comment by revelle** [comment]
I am not suggesting (nor requesting that).  I am just noting that in the latest test versions on win-builder that it now throwing that error.  I assume that win-builder has the latest version. 

I am not calling rBind, lmer seems to be doing so even though it is deprecated.

Thanks.

Bill


> On Mar 21, 2018, at 8:53 PM, Ben Bolker <notifications@github.com> wrote:
> 
> rBind has been deprecated since R version 3.2.0. Should we try to maintain backward compatibility?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
> 

William Revelle		           personality-project.org/revelle.html
Professor			                   personality-project.org
Department of Psychology         www.wcas.northwestern.edu/psych/
Northwestern University	           www.northwestern.edu/
Use R for psychology                 personality-project.org/r
It is 2 1/2  minutes to midnight	   www.thebulletin.org

**Comment by bbolker** [comment]
that question was aimed at the other lme4 authors ...

**Comment by mmaechler** [comment]
I'm going to look into this.
Win-builder got the latest version of Matrix yesterday, (1.2-13), and that *finally* did signal the deprecation -- i.e. a lapsus in earlier versions of Matrix had the deprecation only in the documentation for all practical purposes.     lme4 definitely should not call rBind() [or cBind()] anymore.    I thought I did check lme4 with that version of 'Matrix' before releasing it (Matrix) to CRAN .. but the check must have accidentally used a "released" version of Matrix instead, so the problem did not trigger.

**Comment by mmaechler** [comment]
Closed via 9d5d433d40408222b290d2780ab6e9e4cec553b9
